src/stylesheets/components/blog/_article.scss

```diff
@@ -0,0 +1,152 @@
+.article-inner__titleCard {
+  background-color: white;
+  margin: 0 auto rem(58) auto;
+  margin-bottom: rem(58);
+  max-width: 1200px;
+  padding-top: rem(23);
+  width: 100%;
+
+  h1 {
+    font-weight: 300;
+    margin: 0 auto;
+    margin-bottom: 3rem;
```

Comment:
'Sigh', this is a very bad habit of mine where I apply the ```margin: 0 auto;```  then I override it with another...